tests/foreman/ui_airgun/test_hostcollection.py

```diff
@@ -635,3 +702,89 @@ def test_negative_hosts_limit(session, module_org, module_loc):
         assert "cannot have more than 1 host(s) associated with host collection '{}'".format(
             hc_name
         ) in str(context.value)
+
+
+@tier3
+@upgrade
+def test_positive_install_module_stream(
+        session, module_org, vm_content_hosts_module_stream, vm_host_collection_module_stream):
+    """Install a module-stream to hosts inside host collection remotely
+
+    :id: e5d882e0-3520-4cb6-8629-ef4c18692868
+
+    :Steps:
+        1. Running dnf upload profile to sync module streams from hosts to Satellite
+        2. Navigating to host_collection
+        3. Installing the module stream duck
+        4. Verifying that remote job get passed
+        5. Verifying that package get installed
+
+    :expectedresults: Module-Stream was successfully installed on all the hosts
+        in host collection
+
+    :CaseLevel: System
+    """
+    with session:
+        _run_remote_command_on_content_hosts(
+            'dnf -y upload-profile',
+            vm_content_hosts_module_stream
+        )
+        session.organization.select(org_name=module_org.name)
+        result = session.hostcollection.manage_module_streams(
+            vm_host_collection_module_stream.name,
+            action_type="Install",
+            module_name=FAKE_3_CUSTOM_PACKAGE_NAME,
+            stream_version="0"
+        )
+
+        assert result['overview']['job_status'] == 'Success'
+        assert result['overview']['job_status_progress'] == '100%'
+        assert int(result['overview']['total_hosts']) == 2
+        assert _is_package_installed(vm_content_hosts_module_stream, FAKE_3_CUSTOM_PACKAGE)
+
+
+@tier3
+@upgrade
+def test_positive_install_modular_errata(
+        session, module_org, vm_content_hosts_module_stream, vm_host_collection_module_stream):
+    """Install Modular Errata generated from module streams.
+
+    :id: 8d6fb447-af86-4084-a147-7910f0cecdef
+
+    :Steps:
+        1. Generated errata by installing and downgrading the Module Stream Kangaroo
```

Comment:
Technically, you don't downgrade the module, just install it in outdated version. Or am I missing something?

Reply:
Earlier I did installed module stream and downgrade module stream, later @vijay8451 was asking to update an older version rather downgrading. I missed updating that line, I will update that.